includes/classes/prod_cat_pulldown.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+	class prod_cat_pulldown extends base
+	{
+		var $pulldown, $attributes, $order_by;
+		/**
+		 * @var bool
+		 */
+		private $show_id;
+		/**
+		 * @var array
+		 */
+		private $exclude;
+
+		function __construct(
+			string $pulldown = 'product',
+			array $exclude = [],
+			string $keywords = '',
+			string $order_by = '',
+			bool $attributes = false,
+			$filter_by_option_name = null
+		) {
+
+			$this->exclude = $exclude;
+
+			$this->show_model = true;
```

Comment:
Might think that ultimately this would be controlled by something else either a session variable or admin switch... May help reduce support questions (though of course also add some in any case).

Reply:
see comment above.

Reply:
Agree.. Didn't really want to comment on this at this stage of development, but also didn't want it to get fully lost...